src/coffee/providers/map-loader.coffee

```diff
@@ -34,6 +38,25 @@ angular.module('uiGmapgoogle-maps.providers')
       isGoogleMapsLoaded = ->
         angular.isDefined(window.google) and angular.isDefined(window.google.maps)
 
+      onWindowLoad = (options)->
+        # if its a WebView
+        if !(!window.cordova && !window.PhoneGap && !window.phonegap && !window.forge)
+          document.addEventListener 'deviceready', ->
+            # Cordova specific https://github.com/apache/cordova-plugin-network-information/
+            if window.navigator.connection && window.Connection && window.navigator.connection.type == window.Connection.NONE
```

Comment:
I think this function should not be named isCordova because is checking for network information objects that actually could be injected by another webview distinct from cordova.